src/components/table/table-header-filters/TableHeaderFilters.tsx

```
@@ -37,18 +46,25 @@
+          tableFilters: tableFilters,
+          tableFilter: tableFilter,
+          addFilters: addFilters,
+          removeFilters: removeFilters,
+          resetFilters: resetFilters,
+          filterDispatch: filterDispatch,
+        }
+
+        if (filter.type === FILTER_TYPE.DATE_RANGE) {
+          return <TableHeaderDateRangeFilter {...filterProps} />
+        }
+        return <TableHeaderFilter {...filterProps} />
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)